src/services/sessions.js

```diff
@@ -3199,7 +3199,7 @@ module.exports = class SessionsHelper {
 
 	static async validateMentorExtensions(userIds) {
 		try {
-			const filteredUserIds = userIds.filter((id) => typeof id === 'number')
+			const filteredUserIds = userIds.filter((id) => typeof id === 'number' || typeof id === 'string')
```

Comment:
@rakeshSgr what if we don't add this filter,anyways number or string will cover everything?